plugins/azure/appConfigurations/appConfigurationManagedIdentity.spec.js

```diff
@@ -0,0 +1,118 @@
+var expect = require('chai').expect;
+var appConfigurationManagedIdentity = require('./appConfigurationManagedIdentity.js');
+
+const appConfigurations = [
+    {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "location": "eastus",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2023-12-27T09:26:54+00:00",
+          "endpoint": "https://meerab-test-rg.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": null
+          },
+          "privateEndpointConnections": null,
+          "publicNetworkAccess": "Disabled",
+          "disableLocalAuth": false,
+          "softDeleteRetentionInDays": 0,
+          "enablePurgeProtection": false
+        },
+        "id": "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/meerab-rg/providers/Microsoft.AppConfiguration/configurationStores/meerab-test-rg",
+        "name": "meerab-test-rg",
+        "tags": {}
+    },
+    {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "location": "eastus",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2023-12-27T09:26:54+00:00",
+          "endpoint": "https://meerab-test-rg.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": null
+          },
+          "privateEndpointConnections": null,
+          "publicNetworkAccess": "Disabled",
+          "disableLocalAuth": false,
+          "softDeleteRetentionInDays": 0,
+          "enablePurgeProtection": false
+        },
+        "id": "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/meerab-rg/providers/Microsoft.AppConfiguration/configurationStores/meerab-test-rg",
+        "name": "meerab-test-rg",
+        "tags": {},
+        "identity": {
+            "type": "systemassigned,userassigned",
+            "principalId": "dc03d47d-e6df-491f-aebe-50a93412a890",
+            "tenantId": "d207c7bd-fcb1-4dd3-855a-cfd2f9b651e8",
+            "userAssignedIdentities": {
+              "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourcegroups/meerab-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/testmeerab": {
+                "PrincipalId": "1d34c2cd-bd53-487d-b3a9-6064465497c9",
+                "ClientId": "2071caa1-3668-4de3-babc-155cfe3e38e5"
+              }
```

Comment:
```suggestion
            "principalId": "dc03d47d-e6df-491f-aebe-50a93412a890",
            "tenantId": "d207c7bd-fcb1-4dd3-855a-cfd2f9b651e8",
            "userAssignedIdentities": {
              "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourcegroups/meerab-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/testmeerab": {
                "PrincipalId": "1d34c2cd-bd53-487d-b3a9-6064465497c9",
                "ClientId": "2071caa1-3668-4de3-babc-155cfe3e38e5"
              }
```
make sure we're using dummy values here